run_dir/design/deliveries.html

```diff
@@ -25,7 +25,7 @@
   <div id="ongoing_deliveries">
     <h1>Ongoing <span id="page_title">Deliveries</span></h1>
```

Comment:
Nothing to do with your code, but since you're working on this now, maybe I can inject some style change request 😈 

Can we add some margin here:

`<h1 class="mt-4">Ongoing <span id="page_title">Deliveries</span></h1>`
`<div class="row mt-5">`